daemon/loop.go

```diff
@@ -21,8 +21,24 @@ const (
 
 // Loop for potentially long-running stuff. This includes running
 // jobs, and looking for new commits.
-
 func (d *Daemon) Loop(stop chan struct{}, logger log.Logger) {
+	logger.Log("user", d.GitConfig.UserName,
+		"email", d.GitConfig.UserEmail,
+		"sync-tag", d.GitConfig.SyncTag,
+		"notes-ref", d.GitConfig.NotesRef)
+
+	for d.Checkout == nil {
```

Comment:
Doing the checkout in a while loop here feels a bit wierd.
> Temporary bodge.

Probably best to put a TODO in to look at this again.